<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Camera_CatalogyClass</class>
 <widget class="QMainWindow" name="Camera_CatalogyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera_Catalogy</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Название модели</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Категория</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Производитель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Разрешение матрицы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Габариты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Вес</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Цена</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Дата выпуска</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_ModelName">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="maxLength">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_Category">
        <item>
         <property name="text">
          <string>Профессиональный</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Любительский</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Полупрофессиональный</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_Producer">
        <item>
         <property name="text">
          <string>Nicon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Panasonic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sony</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Canon</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Olympus</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Зенит</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_CamMatrix">
        <property name="minimum">
         <double>2.000000000000000</double>
        </property>
        <property name="maximum">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Dimensions">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_Weight">
        <property name="minimum">
         <number>100</number>
        </property>
        <property name="maximum">
         <number>8000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_Price">
        <property name="minimum">
         <number>1000</number>
        </property>
        <property name="maximum">
         <number>150000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="checkBox_LensOptions">
        <property name="text">
         <string>Возможности смены объектива</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_IsDigital">
        <property name="text">
         <string>Цифровой</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButton_SaveInfo1">
        <property name="text">
         <string>Сохранить запись 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_ShowInfo1">
        <property name="text">
         <string>Показать запись 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_ShowInfo2">
        <property name="text">
         <string>Показать запись 2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButton_SaveInfo2">
        <property name="text">
         <string>Сохранить запись 2</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Camera_Catalogy.qrc"/>
 </resources>
 <connections/>
</ui>
